doc/faq.rst

```diff
@@ -26,6 +26,20 @@ See :ref:`contributing`. Before wanting to add a new algorithm, which is
 usually a major and lengthy undertaking, it is recommended to start with :ref:`known
 issues <easy_issues>`.
 
+What's the best way to ask questions about scikit-learn?
+--------------------------------------------------------------
+We of course want to help scikit-learn users. But for broader questions about
+using the package or about machine learning, we suggest searching 
+`Stack Overflow <http://stackoverflow.com>`_ or 
+`Cross Validated <http://stats.stackexchange.com>`_ before contacting the mailing
```

Comment:
is the scikit-learn list even an appropriate venue for questions about general machine learning...?